nix/examples/simple-codegen/src/Main.purs

```diff
@@ -0,0 +1,3 @@
+module Example.Simple.Main where
+
+import Generated (answer)
```

Comment:
```suggestion
import Generated (answer)

```

Reply:
We should ensure newlines even in generated files. Other than this :+1:

Reply:
Done!